logsink/logsink.hpp

```diff
@@ -40,6 +40,23 @@ class FileSink : public google::LogSink
 
   ~FileSink();
 
+  virtual void send(
+      google::LogSeverity severity,
+      const char* full_filename,
+      const char* base_filename,
+      int line,
+      const struct ::tm* tm_time,
+      const char* message,
+      size_t message_len,
+      google::int32 usecs);
+
+  // This method implements the now obsolete LogSink interface without the
+  // `usecs` argument introduced by a glog patch in MESOS-9687.
+  //
+  // At the moment of the writing defining this method is still necessary due
+  // to the need to maintain compatibility between glog and the old LogSink
+  // implementations. (In the base class this method is purely virtual,
+  // and the new sink() is implemented there in the terms of the obsolete one.)
```

Comment:
I think it's accurate to say the reverse: "the obsolete sink() is implemented there in terms of the new one.", since the older obsolete method now simply calls into the new one.

Reply:
Right now the situations in the base and in the new derived class are exactly opposite to each other: `LogSink::send(..., usecs)` is implemented in terms of `LogSink::send() = 0`, but `FileSink::send()` is defined in terms `FileSink::send(..., usecs)`

But I hope this will change at some point.

I decided to remove this sentence altogether to avoid confusing readers.